src/test/scala/Tests.scala

```diff
@@ -3,39 +3,174 @@ import java.io.File
+      "delete",
+      "id=1"
+    )
+
+    val strSeq3: Seq[String] = Seq(
+      "delete",
+      "--id=-1"
+    )
+
+    val strSeq4: Seq[String] = Seq(
+      "delete",
+      "--id=1"
+    )
+
+    //Error message should appear since --delete is the wrong command, should just be delete
+    Parser.apply(strSeq1)
+
+    //Error message should appear since id=1 is the wrong command, should be --id=1
+    Parser.apply(strSeq2)
+
+    //Error message should appear since --id=-1 has an id of less than 0 when it should be >= 0
+    Parser.apply(strSeq3)
+
+    //Correct format
+    Parser.apply(strSeq4)
+  }
+
+  it should "check to see if the list command is using the correct format" in {
+
+    val strSeq1: Seq[String] = Seq(
+      "--list"
+    )
 
+    val strSeq2: Seq[String] = Seq(
+      "list"
+    )
+
+    //Error message because --list is the wront format, should be list
+    Parser.apply(strSeq1)
+
+    //Correct format
     Parser.apply(strSeq2)
   }
-  it should "take in an object and seralize it" in {
-    case class Person(name:String, email:String, age: Int)
-    val nP = new Person("Abenezer1", "@gmail.com", 12)
-    val path = os.pwd/"taskInProgress"
-    writer.searlize(path, "hello2.yaml", nP)
+
+  it should "check to see if the modify command is using the correct format" in {
+
+    val strSeq1: Seq[String] = Seq(
+      "modify",
+      "--id=1",
+      "--name=world --effort=12 --description=hello my name is Justin --due=2022-3-22"
+    )
+
+    val strSeq2: Seq[String] = Seq(
+      "modify",
+      "--id=-1",
+      "--name=world",
+      "--effort=12",
+      "--description=hello my name is Justin",
+      "--due=2022-3-22"
+    )
+
+    val strSeq3: Seq[String] = Seq(
+      "--modify",
+      "--id=1",
+      "--name=world",
+      "--effort=12",
+      "--description=hello my name is Justin",
+      "--due=2022-3-22"
+    )
+
+    val strSeq4: Seq[String] = Seq(
+      "modify",
+      "--id=1",
+    )
+
+    val strSeq5: Seq[String] = Seq(
+      "modify",
+      "--id=1",
+      "--name=world",
+      "--effort=12",
+      "--description=hello my name is Justin",
+      "--due=2022-3-22"
+    )
+
+    //Error message should appear
+    Parser.apply(strSeq1)
+
+    //Error message since id is less than 0
+    Parser.apply(strSeq2)
+
+    //Error message since --modify is wrong format
+    Parser.apply(strSeq3)
+
+    //Correct format without the optional commands
+    Parser.apply(strSeq4)
+
+    //Correct format with optional commands
+    Parser.apply(strSeq5)
+
   }
 
+  it should "Accurately reward the player" in{
+
+    val cal = Calendar.getInstance()
+
+    val mon = new Monster(name="Mom", description="Your mom", toughness=69)
+    val mons = Seq(mon)
 
+    val itm = new Item(name="Butt", description="Hole", affect=6, lifetime=cal)
+    val itms = Seq(itm)
+
+    val status = new StatusEffect(name="Arm", description="Pit", affect=2, lifetime=cal)
+    val statuses = Seq(status)
```

Comment:
There should be tests for Sequences of 0, 1 or many and combinations of these three states for items and status affects.